mindjunkies/forums/tests/test_views.py

```
@@ -254,116 +238,104 @@ def test_post_valid_comment(self):
-        
+
         # Add messages support to request
-        setattr(request, 'session', 'session')
+        setattr(request, "session", "session")
         messages = FallbackStorage(request)
-        setattr(request, '_messages', messages)
-        
+        setattr(request, "_messages", messages)
+
         view = ReplySubmissionView.as_view()
         response = view(
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)